src/command_modules/azure-cli-shell/azclishell/app.py

```diff
@@ -401,12 +401,15 @@ def _special_cases(self, text, cmd, outside):
 
         if cmd.strip() == "quit" or cmd.strip() == "exit":
             break_flag = True
-        elif text.strip() == "clear":  # clears the history, but only when you restart
+        elif text.strip() == "clear-history":  # clears the history, but only when you restart
             outside = True
             cmd = 'echo -n "" >' +\
                 os.path.join(
                     SHELL_CONFIG_DIR(),
                     SHELL_CONFIGURATION.get_history())
+        elif text.strip() == 'clear':
+            outside = True
+            cmd = 'clear'
```

Comment:
As this is now part of the officially CLI repo, this means it also needs to work with CMD.exe (one of our supported shells). Does this? On CMD.exe "clear" isn't a valid command.  It would be 'cls'.